Makefile

```diff
@@ -74,6 +74,7 @@ uninstall: manifests kustomize
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: manifests kustomize
+	cd config/manager && VALIDATOR_IMG=${VALIDATOR_IMG} envsubst < manager.template.yaml > manager.yaml
```

Comment:
Can you extract this template substitution to a separate make target? It will be needed for `make bundle` too.